x/farming/spec/04_messages.md

```diff
@@ -6,60 +6,63 @@ Messages (Msg) are objects that trigger state transitions. Msgs are wrapped in t
 
 ## MsgCreateFixedAmountPlan
 
+This is one of the private plan type messages that anyone can create. A fixed amount plan plans to distribute amount of coins by a fixed amount defined in `EpochAmount`. Internally, `PrivatePlanFarmingPoolAddress` is created and the creator should query the plan's farming pool address and send amount of coins to the farming pool address so that the plan distributes as intended. Note that there is a fee `PlanCreationFee` paid upon plan creation to prevent from spamming attack.
+
 ```go
 type MsgCreateFixedAmountPlan struct {
-    Name               string
-    Creator            string
-    StakingCoinWeights sdk.DecCoins
-    StartTime          time.Time
-    EndTime            time.Time
-    EpochAmount        sdk.Coins
+	Name               string       // name for the plan for display
+	Creator            string       // bech32-encoded address of the creator for the private plan
+	StakingCoinWeights sdk.DecCoins // staking coin weights for the plan
+	StartTime          time.Time    // start time of the plan
+	EndTime            time.Time    // end time of the plan
+	EpochAmount        sdk.Coins    // distributing amount for every epoch
 }
 ```
 
 ## MsgCreateRatioPlan
 
+This is one of the private plan type messages that anyone can create. A ratio plan plans to distribute amount of coins by ratio defined in `EpochRatio`. All coins that the creator has in balances are used. Internally, `PrivatePlanFarmingPoolAddress` is created and the creator should query the plan's farming pool address and send amount of coins to the farming pool address so that the plan distributes as intended. Note that there is a fee `PlanCreationFee` paid upon plan creation to prevent from spamming attack.
```

Comment:
Remove the line because the fees move to newly created `FarmingPoolAddress` and whichever the creator sends to the farming pool address are used. Let me rephrase the sentence.